layouts/contact/list.html

```diff
@@ -75,7 +75,8 @@ <h4>Office Address</h4>
             </div>
           </div>
           <h4>Phone & Email</h4>
-          <p>{{ $address.phone }}<br>
+          <p>
```

Comment:
Should we remove the <br> tag and instead rely on two <p>[...]</p> blocks?

Reply:
I prever keep br. because if remove `<br>` became like this.

![Screenshot_20201101_140359](https://user-images.githubusercontent.com/45872139/97797032-08766080-1c4c-11eb-907b-d841e9041735.png)
![Screenshot_20201101_140415](https://user-images.githubusercontent.com/45872139/97797037-0f9d6e80-1c4c-11eb-9aa8-a636551a06ff.png)